tutorials/manage-zigbee-devices-with-iot/index.mdx

```diff
@@ -0,0 +1,70 @@
+content:
+  h1: Manage Zigbee devices with IoT Hub
+  paragraph: In this tutorial you will learn how to setup and configure Zigbee2mqtt with Scaleway IoT Hub.
+categories:
+  - managed-services
+hero: assets/scaleway-account.png
+tags: iot iot-hub zigbee zigbee2mqtt
+dates:
+  validation: 2022-12-19
+  posted: 2022-12-20
+---
+
+Privacy and security concerns should be at the forefront of any deployment of connected devices used to monitor and/or automate actions in our physical environment. Zigbee protocol is a recognized protocol used to create such mesh network of sensors and actuators in houses and appartements. However, a proprietary gateway to send command and receive information to and from this network is often used. It results in a trade-off on the control over the level of security and on the capabilities for automation and monitoring. This tutorial aims to set the foundation of a fully configurable solution, based on the open source [zigbee2mqtt](https://github.com/Koenkk/zigbee2mqtt) software and the IoT Hub managed service from Scaleway.
+
+[Scaleway IoT Hub](../../managed-services/iot-hub/concepts#iot-hub) is a resilient and MQTT message broker with export capabilities. The use of a message broker will enable us to increase the interoperability of the solution.
+
+# Creating an IoT Hub
+
+1. For testing purposes, you can create an *IoT Hub* (shared plan) following the available [documentation](../../managed-services/iot-hub/quickstart). Dedicated and High availability plans remain recommanded for more critical workloads.
+
+2. Once the Hub is created, you can continue and add a new device with the option "Deny insecure connection" selected. You will need to download <Icon name="download" /> the device certificate and device private key, as well as <Icon name="copy" /> the "Device ID".
+
+  <Lightbox src={["assets/scaleway-configure-device.png","scaleway-retrieve-device-parameters.png"]} size={"medium"} />
+
+3. Finally, you need to download <Icon name="download" /> the certificate used to secure the MQTT endpoint in the *Networks* tabs.
+
+  <Lightbox src={"assets/scaleway-download-hub-certificate.png"} size={"medium"} />
+
+# Installation and configuration of Zigbee2mqtt
+
+Zigbee2mqtt can be installed on a Single Board Computer such as a Raspberry Pi and you should follow first the [install instructions] (https://www.zigbee2mqtt.io/guide/getting-started/#installation). This tutorial assumes that you have a proper installation of Zigbee2mqtt and that your network of connected devices is configured.
+
+## Configuration of MQTT
+
+Zigbee2MQTT is configured using YAML based ```configuration.yaml``` file. You have to modify this configuration file to include the following MQTT settings.
+The ```force_disable_retain``` should be set to ```true``` when using the IoT Hub shared service level because this functionality is not available on this plan.
```

Comment:
```suggestion
<Message type="note">
The ```force_disable_retain``` should be set to ```true``` when using the IoT Hub shared service level because this functionality is not available on this plan.
</Message>
```